tests/test_streaming.py

```diff
@@ -181,6 +175,30 @@ def test_dataloader_single_device(local_remote_dir: Tuple[str, str], batch_size:
         assert len(set(sample_order)) == num_samples
 
 
+@pytest.mark.parametrize('batch_size', [1, 4])
```

Comment:
Minor comment: Since we are just testing exception, we can fixed the other parameters which is not directly impacting the exception such as `batch_size`, `shuffle`, `num_workers` etc. Something like this

```
@pytest.mark.parametrize('batch_size', [4])
@pytest.mark.parametrize('seed', [1111])
@pytest.mark.parametrize('shuffle', [True])
@pytest.mark.parametrize('num_workers', [2])
@pytest.mark.usefixtures('local_remote_dir')
```

Reply:
Cool sounds good